### DISCUSSION TITLE
Facing issue with FEDORA MEDIA WRITER

### DISCUSSION THREAD

**Discussion by partheebankanagaraj** [FedoraQt/MediaWriter#581] [Q&A]
Hi,
I tried to upload the Fedora workstation Distro on my Flash drive through FEDORA MEDIA WRITER V5.0.4 in my windows 11 laptop..
but after writing fedora workstation on my drive. My pendrive is not visible no my laptop to access. I have only option to format it to access the flash drive. I don't find any other option on the  Fedora media writer...

**Comment by grulja** [comment]
You will have to format it first in Fedora Media Writer, otherwise it will not be accessible on Windows as it will not be recognized.
